tracer/src/Datadog.Trace/TraceContext.cs

```diff
@@ -168,7 +164,7 @@ public TimeSpan ElapsedSince(DateTimeOffset date)
             return Elapsed + (_utcStart - date);
         }
 
-        private static void SetSamplingPriority(Span span, SamplingPriority samplingPriority)
+        private static void AddSamplingPriorityTags(Span span, SamplingPriority samplingPriority)
```

Comment:
I renamed this overload to avoid confusion with
```csharp
SetSamplingPriority(SamplingPriority? samplingPriority, bool notifyDistributedTracer = true)
```

Reply:
Makes me wonder if we should move it to `Span` or `SpanExtensions`.

Reply:
> Makes me wonder if we should move it to Span or SpanExtensions.

Seems reasonable to me, given it's nothing to do with `TraceContext`, though `TraceContext` _is_ the only place that uses it, so... 🤷‍♂️

Reply:
I'm gonna keep this PR clear and leave that for later. Thanks, though.